capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -12,6 +12,32 @@ const DOWN_KEY = 'ArrowDown';
 const ENTER_KEY = 'Enter';
 let selectedSuggestionPosition = -1;
 
+/**
+ * Gets suggestions to display to the users
+ */
+function getSuggestionsForUserToChooseFrom() {
+  const val = SEARCH_INPUT.value.trim();
+  if (!val) {
+    return;
+  }
+  const fetchStr = '/search?query=' + val;
+  console.log('Fetch String: ', fetchStr);
+  fetch(fetchStr).then((response) => response.json()).then((suggestions) => {
+    getsAllProposedSuggestions(suggestions, val);
+  });
+}
+
+/**
+ * Sends final college list to the dashboard
+ */
+function sendCollegeInformationToDashboard() {
+  const listItems = document.querySelector('ul').children;
+  const listArray = Array.from(listItems);
+  const idArray = Array.from(listArray, (l) => l.id);
```

Comment:
nit - I think this might be more intuitive with a `map` invocation.

```suggestion
  const idArray = Array.from(listItems).map((l) => l.id).join();
```